docs/howtoguides/get_token_and_attach.md

```diff
@@ -35,3 +46,24 @@ Enable services with: pro enable <service>
 Once the Ubuntu Pro Client is attached to your Ubuntu Pro account, you can use
 it to activate various services, including: access to ESM packages, Livepatch,
 FIPS, and CIS. Some features are specific to certain LTS releases.
+
+## Control of auto-enabled services
+
+Your subscription controls which services are available to you as shown and
+controlled on the [Ubuntu Pro Dashboard](https://ubuntu.com/pro/dashboard).
+
+Recommended services are auto-enabled when attaching a system. Which services
+are considered recommended can, for some subscriptions, be controlled on the
+[Ubuntu Pro Dashboard](https://ubuntu.com/pro/dashboard) by toggling those.
```

Comment:
```suggestion
Recommended services are auto-enabled by default when attaching a system.
You can choose which of the available services will be automatically
enabled or disabled when you attach by toggling them in the
[Ubuntu Pro Dashboard](https://ubuntu.com/pro/dashboard). 
Available services can always be enabled or disabled on the command line
with `pro enable` and `pro disable` after attaching.
```

Reply:
Since you mentioned in the previous paragraph that the subscription controls which services are available, I think this wording is perhaps a bit clearer (and hopefully not too long)